src/Microsoft.DotNet.Darc/DarcLib/ILocal.cs

```diff
@@ -1,18 +1,12 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-using System.Collections.Generic;
 using System.Threading.Tasks;
 
 namespace Microsoft.DotNet.DarcLib;
 
 internal interface ILocal
 {
-    Task UpdateDependenciesAsync(
```

Comment:
why removing from the interface?

Reply:
It actually wasn't used anywhere (the interface) and it is incomplete. I think I will remove the whole interface